src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -4,6 +4,8 @@
     <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;net7.0</TargetFrameworks>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
     <AssemblyVersion>6.0.0.0</AssemblyVersion> <!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <CLSCompliant>false</CLSCompliant>
```

Comment:
```suggestion
```

Reply:
Alright, will remove it. It seems I also need to remove it from Microsoft.IO.Redist. We had that brought in when migrating the assembly.

Reply:
Wait, hold on. Both Microsoft.IO.Redist and System.Runtime.CompilerServices.Unsafe are the two cases that need the property explicitly set to false. They need to stay. If I remove them, then they get the value automatically set to true, and that is not the behavior we want.

Reply:
Ok I didn't know that this property existed there. I'm surprised about CompilerServices.Unsafe though, as it shouldn't matter there as it doesn't use the roslyn compiler but ilasm.